apps/passport-client/components/modals/InvalidUserModal.tsx

```diff
@@ -20,8 +21,15 @@ export function InvalidUserModal() {
         one. Click the button below to log out. Then you'll be able to sync your
         existing Zupass account onto this device.
       </p>
-      <Spacer h={24} />
-      <Button onClick={onClick}>Exit</Button>
+      <Spacer h={16} />
```

Comment:
I can't comment on line 20 above, but it's not accurate.  It says "You've reset your Zupass on another device" which is definitely not the only way you can get into this situation.

Reply:
I've changed it to be more generic.